ui/shared/components/graph/HorizontalStackedBar.jsx

```diff
@@ -21,10 +21,10 @@ const BarContainer = styled.div.attrs(props => ({
   text-align: center;
   border: 1px solid gray;`
 
-const BarSection = styled(({ to, ...props }) => React.createElement(to ? Link : 'div', { to, ...props }))`  
+const BarSection = styled(({ to, ...props }) => React.createElement(to ? Link : 'div', { to, ...props })).attrs(
```

Comment:
if we have a lot of the same bar graph showing different breakdowns, like on the project page where we have a graph for each family's tags, this change means we compute one set of classes used by all of them instead of computing a different set of classes for each graph to make the breakdown render the correct percentages